src/explore.ts

```diff
@@ -85,10 +85,45 @@ async function loadSourceMap(codeFile: File, sourceMapFile?: File): Promise<Sour
   };
 }
 
+function detectEOL(content: string): string {
+  const lineBreakIndex = content.indexOf('\n', 1);
+
+  return content[lineBreakIndex - 1] === '\r' ? '\r\n' : '\n';
```

Comment:
Why not just check `content.contains('\r\n')`?

Reply:
Do you mean `includes`? 
I will replace it with `return content.includes('\r\n') ? '\r\n' : '\n'`